crates/modelardb_server/src/storage/compressed_data_manager.rs

```diff
@@ -350,6 +353,19 @@ impl CompressedDataManager {
 
         Ok(())
     }
+
+    /// Change the amount of memory for compressed data in bytes according to `value_change`. If
+    /// less than zero bytes remain, save compressed data to free memory. If all the data is saved
+    /// successfully return [`Ok`], otherwise return [`IOError`].
+    pub(super) async fn set_compressed_remaining_memory_in_bytes(
```

Comment:
To me it seems a little surprising that `set_...` does not take an absolute value, but rather a `value_change`. I would be less surprised if it was called `adjust...`. But if this isn't surprising to the rest of you or if it is a kind of convention to use `set`, it is fine

Reply:
Good idea, fixed.